lib/generators/acts-as-messageable/migration/migration_generator.rb

```diff
@@ -17,6 +17,7 @@ def self.next_migration_number(dirname)
     def create_migration_file
       migration_template 'migration.rb', 'db/migrate/create_messages_table.rb' rescue nil
       migration_template 'migration_permanent.rb', 'db/migrate/add_recipient_permanent_delete_and_sender_permanent_delete_to_messages.rb' rescue nil
+      migration_template 'migration_opened_as_datetime.rb', 'db/migrate/add_opened_at_to_messages.rb' rescue nil
```

Comment:
Line is too long. [112/80]<br>Avoid using `rescue` in its modifier form.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.